scripts/unit_test_shards_parser.py

```diff
@@ -4,24 +4,32 @@
 import json
 
 
-def get_test_paths_for_shard(shard_name):
+def load_unit_test_shards(shard_name):
     unit_tests_json = f'{os.getcwd()}/.github/workflows/unit-test-shards.json'
```

Comment:
There's no need to put getcwd into this path, the current directory is the default for opening files.